client/src/Components/Questions.js

```diff
@@ -91,7 +91,7 @@ const Questions = (props) => {
 	};
 
 	return (
-		<Form onSubmit={submitHandler} className="survey-form">
+		<Form  className="survey-form">
```

Comment:
I'm afraid that onSubmit is wrong
please don't change unnecessary thing

Reply:
i moved it into the button nex which was empty now you have same thing but onClick in button